# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/HttpChannelChild.cpp
# Commit: aaf5c5213d8d
# Full Hash: aaf5c5213d8db7c9344e8517ae934c55f937e07e
# Author: Christoph Kerschbaumer <ckerschb@christophkerschbaumer.com>
# Date: 2020-10-08 09:49:50
# Regressor Bug: 1665062
# File Overlap Count: 1
# Description:
#   Bug 1665062: HTTPS-Only: Upgraded website creating HTTP auth prompt gets interrupted by error-page r=necko-reviewers,dragana,JulianWels
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D91908
# ==============================================================================

diff -r e4b87a2e0f2c -r aaf5c5213d8d netwerk/protocol/http/HttpChannelChild.cpp
--- a/netwerk/protocol/http/HttpChannelChild.cpp	Tue Oct 06 12:51:16 2020 +0000
+++ b/netwerk/protocol/http/HttpChannelChild.cpp	Wed Oct 07 11:47:07 2020 +0000
@@ -2340,6 +2340,9 @@
   DROP_DEAD();
 }
 
+NS_IMETHODIMP
+HttpChannelChild::GetIsAuthChannel(bool* aIsAuthChannel) { DROP_DEAD(); }
+
 //-----------------------------------------------------------------------------
 // HttpChannelChild::nsICacheInfoChannel
 //-----------------------------------------------------------------------------